README.md

```diff
@@ -111,7 +111,7 @@ HTML/CSS & Javascript
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://feliverse.github.io/conference-page/)
 - [Loom Video Presentation](https://www.loom.com/share/ae12d221fbdf498181028d47d3b9fc13)
```

Comment:
- [ ] Next time please do not merge a branch before it is reviewed, the reviewer needs to have all your changed files in order to well perform the review, apart from that, your work is to salute, nice job @Feliverse, I just have a few issues to raise on the design before the work gets approved

<img width="1201" alt="1" src="https://user-images.githubusercontent.com/77843050/210878076-42ece79d-8152-4092-b974-956324570e5a.png">

- This hero text is not using the same design color and font-weight as per the original design, please try adjusting a bit more your image that serves as text color to match the original design on the left
- Kindly use the correct CSS properties as per the requirement, font, padding, color, and all other staff to get the same design as per the original one

<img width="811" alt="2" src="https://user-images.githubusercontent.com/77843050/210879190-ef51faae-30d9-4799-8496-75374c02e047.png">

- [ ] As I pointed out with arrows, please consider making the necessary changes to match the original design, 
- Let's start with the speaker's name, if I believe my eyes, color, and font weight are not the correct ones using the original template, please fix that too
- The paragraph in red should be styled in `italic` font, please update that and adjust the padding between it and the divider half line. 
- The last text is a bit more black, and the font-weight is confusing a bit, please stick to the design

<img width="928" alt="3" src="https://user-images.githubusercontent.com/77843050/210880618-da05f98f-d844-4cf9-912d-75e0f2fa33a4.png">

- [ ] Please use the same color, font, and padding as per the original design, as shown in the section above, please change the `cc campaign` from black color to red color.

<img width="1248" alt="4" src="https://user-images.githubusercontent.com/77843050/210881271-bea82c47-2125-4c01-a749-afeab9d9bf26.png">

- Please fix this up, the about page text is not the correct one as per the original template
- The paragraph here needs to get a few touches and make look like the one on the original template
- Please make the nav menu wrap the entire page, there is a white space at the bottom of the.

<img width="411" alt="1" src="https://user-images.githubusercontent.com/77843050/210882874-8cc0398a-e2ad-482e-985c-b17e8a463ac2.png">

Reply:
@joskalenda, the recommendations did it, and the page looks much good.
Thank you